source/postprocess/dynamic_topography.cc

```diff
@@ -257,7 +257,9 @@ namespace aspect
                              "in the outputted data file (not visualization). "
                              "'true' subtracts the mean, 'false' leaves "
                              "the calculated dynamic topography as is. ");
-
+          prm.declare_entry ("Density above","0",
+                             Patterns::Double (0),
+                             "The density value out of the surface boundary.");
```

Comment:
I think this is too short an explanation. Can you explain here and in the other place in more detail what this parameter does?